src/web/components/MainMedia.tsx

```diff
@@ -52,6 +54,10 @@ const immersiveWrapper = css`
     overflow: hidden;
 `;
 
+const youtubeWrapperStyles = css`
+	margin-bottom: 12px;
+`;
+
```

Comment:
Is this needed?

Reply:
frontend has a little more spacing between main media and article, I am happy to revert however and check with Harry?

Reply:
The risk is this margin might look right in this context but break thiings in other contexts. MainMedia can be used in lots of places.

Reply:
`Figure` is an attempt to rationalise spacing for in body elements, removing it from individual components and making it so that vertical spacing happens in one place, and one place only. But to implement it we had to review all the places where margins were getting set to make sure we didn't just make the problem worse. It could be we want to use it here for `MainMedia` but we need to do the same review beforehand.